docs/src/app/components/pages/components/List/Page.jsx

```diff
@@ -28,7 +28,7 @@ import ListExampleSelectable from './ExampleSelectable.jsx';
 
 const descriptions = {
   simple: 'A simple `List` with left and right [SVG icons](/#/components/svg-icon).',
-  chat: 'A chat list with Image [Avatars](/#/components/avatar) and List `subheaders`.',
+  chat: 'A chat list with Image [Avatars](/#/components/avatar) and `Subheader`.',
```

Comment:
I supposed we might as well link to the subheader docs since we're doing it with Avatars here. Thoughts?